scripts/211-zoom-autoupdate.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#verificar se usuário está como root
+if [[ $UID != 0 ]]; then
+  echo "Executar este script como root."
+  exit 1
+fi
+
+mkdir /opt/zoom-updater
+
+#copia arquivo zoom-updater para seu respectivo diretorio
+if [ -f "../arquivos/zoom-update.sh" ]; then
+   cp ../arquivos/zoom-update.sh /opt/zoom-updater
```

Comment:
```suggestion
if [ -f "../arquivos/zoom-autoupdate.sh" ]; then
   cp "../arquivos/zoom-autoupdate.sh" /usr/local/cmc/zoom-updater.sh
   chmod +x /usr/local/cmc/zoom-updater.sh
```

No arquivo `arquivos/zoom-update.service` você vai precisar atualizar o caminho do executável para `/usr/local/cmc/zoom-updater.sh`.

Por fim, cê vai precisar adicionar um link simbólico para o `/usr/local/cmc/zoom-updater.sh` lá no script `999`.